swarm/network/stream/common_test.go

```diff
@@ -398,3 +391,55 @@ func (b *boolean) bool() bool {
 
 	return b.v
 }
+
+var (
+	hash0     = sha3.Sum256([]byte{0})
```

Comment:
I just moved these from one file to another, since they are used are at least two places, and because of that I think they belong to `common_test.go`. So I don't think I can scope it in NeedData - I'd have to duplicate it in the other place as well.

But I see your point - input parameters should be close to the test function they are used at, and actually I already fixed that at a few other places. Will review if this can be improved as well.